package/yast2-drbd.spec

```diff
@@ -31,46 +34,43 @@ BuildRequires:  update-desktop-files
 BuildRequires:  yast2 >= 4.0.39
 BuildRequires:  yast2-devtools >= 3.1.10
```

Comment:
%yast_metainfo is added in 4.2.2, so dependencies of yast2-devtools should update.